build-system/tasks/prepend-global/index.js

```diff
@@ -140,6 +142,7 @@ function applyConfig(
         if (opt_localDev) {
           configJson = enableLocalDev(config, target, configJson);
         }
+        configJson.test = !!opt_fortesting;
```

Comment:
Two things:
- I think it's safer to add `test` to `configJson` only when `opt_fortesting` is `true`
- When `true`, use `Object.assign()` with `{ test: true }`

Reply:
Done